DenunciaCiudadana/src/screens/UserSettings.js

```diff
@@ -0,0 +1,62 @@
+import React, {useState} from 'react';
+import {View} from 'react-native';
+import FormTitle from '../components/FormTitle';
+import FormText from '../components/FormText';
+import FormInput from '../components/FormInput';
```

Comment:
You can create a general file in components with name index.js and import and export all your component from there...